src/pfs/pfs.proto

```diff
@@ -114,8 +114,10 @@ message CommitOrigin {
 // protos)
 message Commit {
   option (gogoproto.goproto_stringer) = false;
+  Repo repo = 3;
+  // deprecated
```

Comment:
Should this get the deprecated field option? `[deprecated = true]`

If so, we should consider whether the first PR after a release should go through and change all deprecated fields to reserved ones.